volunteers/models.py

```diff
@@ -1,17 +1,8 @@
 from django.db import models
 from django.core.validators import MinLengthValidator
+from django.forms import ModelForm
```

Comment:
I think this needs to be reserve for ModelForm

Reply:
Changed that in my newest PR.